def summa (nmbr1, nmbr2):
    try:
        nmbr1 =  __add__(self, nmbr2)
    except ValueError:
        return("Неверное значение")
    return(nmbr1)
      